jena-core/src/test/java/org/apache/jena/test/TestPackage.java

```diff
@@ -32,7 +33,7 @@
 public class TestPackage extends TestCase {
 
```

Comment:
It's because I `@Deprecate`d `rdfcat`. I'll suppress those warnings, since they convey no real information.